source/simulator/stokes_matrix_free.cc

```diff
@@ -755,6 +792,31 @@ namespace aspect
               for (unsigned int d=0; d<dim; ++d)
                 sym_grad_u[d][d] -= viscosity_x_2/3.0*div;
 
+            if (strain_rate_table !=nullptr
+                && !(strain_rate_table->empty()))
+              {
+                SymmetricTensor<2,dim,VectorizedArray<number>> grads_phi_u_i =
+                                                              velocity.get_symmetric_gradient (q);
+                // deta_deps_times_eps_times_phi[i]
+                // = (viscosity_derivative_wrt_strain_rate * scratch.grads_phi_u[i]) * strain_rate;
+
+                // derivative_scaling_factor * alpha *
```

Comment:
maybe we should delete this? or do you think you can clean it up to be more helpful?

Reply:
I removed them.